spec/workers/modify_project_updated_at_worker_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe ModifyProjectUpdatedAtWorker do
+    let(:worker) { described_class.new }
+    let(:project) { create :project }
+
+    describe 'project touch timestamp' do
+        before do
+          allow(Project).to receive(:find).and_return(project)
+        end
+    
+        it 'touches the project timestamp on update' do
+          expect(project).to have_received(:touch).once
+          worker.perform(project)
```

Comment:
You'll need to pass in the id not the active record resource.
```suggestion
          worker.perform(project.id)
```